pkgs/top-level/perl-packages.nix

```diff
@@ -17825,6 +17825,20 @@ let
     };
   };
 
+  NetIPXS = buildPerlPackage {
+    pname = "Net-IP-XS";
+    version = "0.22";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/T/TO/TOMHRR/Net-IP-XS-0.22.tar.gz";
+      hash = "sha256-JZe0aDizgur3S6XJnD9gpqC1poHsNqFBchJL9E9LGSA=";
+    };
+    propagatedBuildInputs = [ IOCapture MathBigInt TestMore TestPod TieSimple ];
```

Comment:
```suggestion
    propagatedBuildInputs = [ IOCapture MathBigInt TestPod TieSimple ];
```

Running `nixpkgs-review pr 214925` fails as it's unable to find `TestMore` (for some reason).

In any case, Test::More is a part of perl core so it doesn't need to be listed as a dependency.

```
Automatic merge went well; stopped before committing as requested
error: undefined variable 'TestMore'

       at /home/sgo/.cache/nixpkgs-review/pr-214925-2/nixpkgs/pkgs/top-level/perl-packages.nix:17835:52:

        17834|     };
        17835|     propagatedBuildInputs = [ IOCapture MathBigInt TestMore TestPod TieSimple ];
             |                                                    ^
        17836|     meta = {
(use '--show-trace' to show detailed location information)
nix --experimental-features nix-command --system x86_64-linux eval --json --impure --no-allow-import-from-derivation --expr (import /nix/store/28x296plr3khwyi163wgbyw77kw50v26-nixpkgs-review-2.8.0/lib/python3.10/site-packages/nixpkgs_review/nix/evalAttrs.nix { allowAliases = false; attr-json = /run/user/1000/tmpy6jtjl7k; }) failed to run, /run/user/1000/tmpy6jtjl7k was stored inspection
https://github.com/NixOS/nixpkgs/pull/214925 failed to build
```